code/datums/brain_damage/hypnosis.dm

```diff
@@ -0,0 +1,43 @@
+/datum/brain_trauma/hypnosis
+	name = "Hypnosis"
+	desc = "Patient's unconscious is completely enthralled by a word or sentence, focusing their thoughts and actions on it."
+	scan_desc = "looping thought pattern"
+	gain_text = ""
+	lose_text = ""
+	resilience = TRAUMA_RESILIENCE_SURGERY
+	
+	var/hypnotic_phrase = ""
+	
+/datum/brain_trauma/hypnosis/New(phrase)
+	hypnotic_phrase = phrase
+	if(!phrase)
+		qdel(src)
+	..()	
+	
+/datum/brain_trauma/hypnosis/on_gain()
+	message_admins("[ADMIN_LOOKUPFLW(owner)] was hypnotized with the phrase '[hypnotic_phrase]'.")
+	log_game("[key_name(owner)] was hypnotized with the phrase '[hypnotic_phrase]'.")
+	to_chat(owner, "<span class='big'>[hypnotic_phrase]</span>")
+	to_chat(owner, "<span class='notice'>[pick("You feel your thoughts focusing on this phrase... you can't seem to get it out of your head.",\
+												"Your head hurts, but this is all you can think of. It must be vitally important.",\
```

Comment:
Trances often don't induce migraines on their own.

Reply:
Yeah i'm using a fairly inaccurate version of hypnosis here, i didn't prioritize realism